programs-remote-tests/composer.json

```diff
@@ -1,11 +1,12 @@
 {
-    "require": {},
     "require-dev": {
-        "codeception/codeception": "3.1.3",
+        "codeception/codeception": "^4.1",
         "captbaritone/mailcatcher-codeception-module": "1.*",
         "squizlabs/php_codesniffer": "2.7.*",
         "wimg/php-compatibility": "*",
-        "simplyadmire/composer-plugins" : "@dev"
```

Comment:
I guess were not using this anymore

Reply:
Doesn't appear so, and it's an abandoned package, so I just dropped it.